script/start_server.sh

```
@@ -1,3 +1,6 @@
 #!/bin/bash
-cd /home/ubuntu/build
-pm2 start ../apps/api/src/index.js
+# express 실행
+cd /home/ubuntu/build/codeit-resources
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)